### PR TITLE
Thank you mail refined

### PR DESCRIPTION
**Pull request by nickhsine**
### Change Log
- Format template/success-donation.tmpl
- Update template/success-donation.tmpl. Add do-not-reply message
- Thank you mail title changed
- Email sender named changed: `no-reply@twreporter.org` -> `報導者 The Reporter <no-reply@twreporter.org>`

### CODE REVIEW

**Comment by babygoat** [comment]
1. Is there any style guide for *.tmpl?
2. Please attach the link for the requested changes.

**Comment by nickhsine** [comment]
for 2:
please check the response from editors.
https://paper.dropbox.com/doc/--AZgKjA7iGeGGk3q~rEu3SY2dAg-u0HAqpZnNn3gxV6TDvEfm#:uid=780469015880172074351782&h2=%E4%BF%A1%E4%BB%B6%E4%B8%BB%E6%97%A8%E3%80%81%E5%85%A7%E5%AE%B9%E8%AA%BF%E6%95%B4

for 1: 
I haven't found a better way to format the `.tmpl` file.
I will be still searching the approach.

**Comment by nickhsine** [comment]
```
寄件者名稱
no-reply → 報導者 The Reporter
```
The above request is still in the progress.
I will commit another patch in this PR.

**Comment by babygoat** [comment]
> for 1:
> I haven't found a better way to format the `.tmpl` file.
> I will be still searching the approach.
If this is not determined, I might suggest leaving only the changes for required rephrase. For one thing, this would be simple to review. For another, the following changes to the *.tmpl would not be repetitive addressing the style changes before the style guides is draft or determined.

**Comment by nickhsine** [comment]
updated.

File: configs/config.go [hunk 1]
```diff
@@ -110,9 +111,10 @@ type SMTPConfig struct {
 }
 
 type AmazonConfig struct {
-	Sender    string `yaml:"sender"`
-	AwsRegion string `yaml:"aws_region"`
-	Charset   string `yaml:"char_set"`
+	SenderMail string `yaml:"sender_mail"`
```
Comment (babygoat) [inline configs/config.go]: Minor. SenderAddress would be better?

File: services/mail.go [hunk 2]
```diff
@@ -35,13 +36,21 @@ type AmazonMailStrategy struct {
 	conf configs.AmazonConfig
 }
 
+func (s *AmazonMailStrategy) MIMEForEmailTitle(charSet, title string) string {
+	const encoding string = "B" // base64
+	var encodedText = base64.StdEncoding.EncodeToString([]byte(title))
+	return fmt.Sprintf("=?%s?%s?%s?=", charSet, encoding, encodedText)
```
Comment (babygoat) [inline services/mail.go]: Reminder: encoded-word syntax(rfc-2047)
https://en.wikipedia.org/wiki/MIME#Encoded-Word

File: template/success-donation.tmpl [hunk 3]
```diff
@@ -1,1105 +1,1508 @@
...
+												</tbody>
+											</table>
+											<table align="left" border="0" cellpadding="0" cellspacing="0" style="max-width:100%; min-width:100%;" width="100%" class="mcnTextContentContainer">
+												<tbody>
+													<tr>
+
+														<td valign="top" class="mcnTextContent" style="padding: 0px 18px 9px;color: #9C9C9C;line-height: 100%;">
+															<p style="font-size: 18px !important;color: #9C9C9C;line-height: 100%;">
+																<span style="font-size:13px;color:#888888;font-family:source sans pro,helvetica neue,helvetica,arial,sans-serif">*本信件由系統自動發出，請勿直接回覆！*</span>
+															</p>
```
Comment (babygoat) [inline template/success-donation.tmpl]: Do we need a separation line above as in the change request document?

**Comment by babygoat** [comment]
LGTM

File: template/success-donation.tmpl (hunk 3 above)
Comment (nickhsine) [reply template/success-donation.tmpl]: The CSS will generate the separation line.

**Comment by nickhsine** [comment]
Updated.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,13 @@
 ## Released
 
+### 3.0.3
+#### Thank you mail refinement
+- Format template/success-donation.tmpl
+- Update template/success-donation.tmpl. Add do-not-reply message
+- Thank-you mail title changed
```
Comment (babygoat) [inline CHANGELOG.md]: minor. `changed`  -> `revised`.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,13 @@
...
+- Email sender name changed: `no-reply@twreporter.org` -> `報導者 The Reporter <no-reply@twreporter.org>`
+- Thank you mail wording change: 捐款 -> 贊助
```
Comment (babygoat) [inline CHANGELOG.md]: `Thank you mail`  or `Thank-you mail`?

**Pull request merged**